src/diffusers/models/attention.py

```diff
@@ -188,7 +188,11 @@ def set_chunk_feed_forward(self, chunk_size: Optional[int], dim: int = 0):
         self._chunk_dim = dim
 
     def forward(
-        self, hidden_states: torch.FloatTensor, encoder_hidden_states: torch.FloatTensor, temb: torch.FloatTensor
+        self,
+        hidden_states: torch.FloatTensor,
+        encoder_hidden_states: torch.FloatTensor,
+        temb: torch.FloatTensor,
+        joint_attention_kwargs: Dict[str, Any] = None,
```

Comment:
```suggestion
        joint_attention_kwargs: Dict[str, Any] = {},
```

Or we can check `joint_attention_kwargs` is not None before unpacking

Reply:
Thanks for review😊. I have committed this fix.